libs/execution_context/ebpf_program.c

```diff
@@ -8,6 +8,7 @@
 #include "ebpf_link.h"
 #include "ebpf_object.h"
 #include "ebpf_program.h"
+#include "ebpf_program_attach_type_guids.h"
```

Comment:
Line 229 etc has "uuid" but above has "guid".  Pick one and be consistent.  I believe UUID is the IETF RFC term.

Reply:
Renamed functions.